qemu.go

```diff
@@ -106,6 +108,11 @@ const (
 	maxDevIDSize = 31
 )
 
+const (
+	// NVDIMM device needs memory space (1024MB is suggested by @grahamwhaley)
```

Comment:
something like this? https://github.com/clearcontainers/runtime/issues/380#issuecomment-321216520